tests/beaker_tests/acl/test_acl.rb

```diff
@@ -0,0 +1,271 @@
+###############################################################################
```

Comment:
@mwiebe Just leaving a breadcrumb to fix this up post-merge: `cd tests/beaker_tests ; git mv acl cisco_acl`